### ISSUE TITLE
Icon Pack Support For Launchers

### ISSUE THREAD

**Issue by GitHG22** [beli3ver/Paper-FOSS-Theme#19]
Will you consider adding support for launchers like Nova, Action Launcher, etc.?

**Comment by beli3ver** [comment]
Yes I will, but I could not say one. Maybe with the first stable release.

**Comment by dkanada** [comment]
I can help with the icon pack if you still want to add support for launchers. It seems like the the repository on gitlab is different, are they different projects or is one of the repos just a little out of date?

**Comment by beli3ver** [comment]
Thanks.
I forked yours and created a new one:

https://gitlab.com/beli3ver/Paper-FOSS-Theme

Why you stoped with your theme?

**Comment by dkanada** [comment]
I haven't stopped, just been focusing on other projects for a while. I really want to come up with a way to pull the icons out of the app so more open source icon packs can use it if they want to. Ideally the icon pack resources would be kept in a separate repo and the app itself would have an option to compile with the resources included. The desired result is more people submitting icon packs to F-Droid, there aren't very many there. I also thought creating an organization on github for open source icon packs might spur development. That way if someone had experience with graphic design they wouldn't need to know how to code, they could just find a repo and start creating icons.

**Comment by beli3ver** [comment]
THat's sound great.
Maybe like this, greate a organisation,

***OpenFreeIcons*** (something like that)

Then create a BaseRepo for the main icons. And then a repo for every theme, Android, Desktop etc...
Nice idea.
So you could use the Basic icons in many Themes, and you/we can create many themes an more people will use the icons....

**Comment by conrad-heimbold** [comment]
@dkanada @beli3ver 

A Free SVG icon collection that can easily be changed and used in other icon packs 
is **exactly** what I've been doing, check out my repo conrad-heimbold/SimpleVectorIcons! 
That repo **only contains graphical assets** (svg, png, avd), **no code** at all (except of some scripts to handle the svgs). 

- You can change the colors for every icon just by editing the css stylesheet. 
- I also separated the icons in my repo into 
    - "more general category icons", which represent a general physical object / idea that doesn't apply to a specific app/service only by its shape; and 
    - "specific trademark/logo/emblem" icons, whose shape only makes sense for a specific app/company/service. 
- Every icon is divided into a "foreground" and "background" part. You can replace the background or foreground part separately with whatever you want/whatever looks better.

**Comment by dkanada** [comment]
The goal is more unique icons for the apps on F-Droid and not just recolored icons, although that is a really cool project. The paper icon pack is an excellent example, the designs are conpletely different from ICEcons and are also very colorful. Some people like icons that are uniformly shaped, monotone, or with lots of other variations that can share a similar base but usually have wildly different designs in the end. Hopefully if people had a complete appfilter for all the apps on F-Droid and a simple app they could use for their icons they would spend more time designing the icons themselves.

**Comment by dkanada** [comment]
The goal is completely different icons that have their own style though, not simply recolored icons, although that is a very cool idea. Some people like icons that are uniformly shaped, monotone, or with lots of other variations that would require a custom design. Hopefully if people had a complete appfilter for the apps on F-Droid and a simple app they could use for their icons they would be able to focus more on designing the icons themselves.

**Comment by dkanada** [comment]
Just saw your other comment in the pull request and I agree. If we split everything into separate repos then I would suggest one repo for the app itself, separate repos for any icon pack that uses the app, and one more repo for extra data such as the appfilter. We should make sure this is a good idea before breaking everything up though.

The extra repo would definitely contain the appfilter, drawable, and iconpack xml files. Those files would contain the data for every icon published to the main repos. I also think it should contain the "foreground" sections from your repo as templates if other people want to use them.

**Comment by beli3ver** [comment]
@conrad-heimbold I love your theme.
I will directly create a new F-Droid Icon theme with it.
I will use the white one, and change the color to #646464

**Comment by dkanada** [comment]
Wait @beli3ver I think @conrad-heimbold is making an app that will automatically change the icon to any color, so your icon pack might be redundant.

**Comment by conrad-heimbold** [comment]
@dkanada I still need a lot of time to achieve that (customizable color inside the app); and don't know enough of Android coding yet; so doing in manually with a separate app is faster at the moment.
